test_log1 (logs_test.TestLogs) ... ok
test_log2 (logs_test.TestLogs) ... ok
test_exec_message_format (message_test.TestMessage) ... ok
test_header_gets (message_test.TestMessage) ... ok
test_header_sets (message_test.TestMessage) ... ok
test_log_on_gets (message_test.TestMessage) ... ok
test_log_on_sets (message_test.TestMessage) ... ok
test_order_message_format (message_test.TestMessage) ... ok
test_trade_message_gets (message_test.TestMessage) ... ok
test_trade_message_sets (message_test.TestMessage) ... ok
test_trailer_gets (message_test.TestMessage) ... ok
test_trailer_sets (message_test.TestMessage) ... ok
test_add_order_to_pool (order_pool_test.TestOrderPool) ... ok
test_serach_order_pool (order_pool_test.TestOrderPool) ... ok
test_update_order_to_pool (order_pool_test.TestOrderPool) ... ok
test_order_gets (order_test.TestOrder) ... ok
test_order_headers (order_test.TestOrder) ... ok
test_order_print_order (order_test.TestOrder) ... ok
test_order_set_position (order_test.TestOrder) ... ok
test_order_value (order_test.TestOrder) ... ERROR
test_reader_create_mess_heart (reader_test.TestReader) ... ERROR
test_reader_create_mess_logon (reader_test.TestReader) ... ERROR
test_reader_create_mess_trade (reader_test.TestReader) ... ERROR
test_reader_read_line (reader_test.TestReader) ... ERROR

======================================================================
ERROR: test_order_value (order_test.TestOrder)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/joanna/Desktop/jobs/OnGoing/quantifeed/unit_test/order_test.py", line 60, in test_order_value
    self.order.order_aalue())
AttributeError: 'Order' object has no attribute 'order_aalue'

======================================================================
ERROR: test_reader_create_mess_heart (reader_test.TestReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/joanna/Desktop/jobs/OnGoing/quantifeed/unit_test/reader_test.py", line 24, in test_reader_create_mess_heart
    self.assertEqual(58, mes.get_header().getBodyLength())
AttributeError: 'Header' object has no attribute 'getBodyLength'

======================================================================
ERROR: test_reader_create_mess_logon (reader_test.TestReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/joanna/Desktop/jobs/OnGoing/quantifeed/unit_test/reader_test.py", line 33, in test_reader_create_mess_logon
    mes = self.reader.createMessage(mes_log_on)
AttributeError: 'Reader' object has no attribute 'createMessage'

======================================================================
ERROR: test_reader_create_mess_trade (reader_test.TestReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/joanna/Desktop/jobs/OnGoing/quantifeed/unit_test/reader_test.py", line 42, in test_reader_create_mess_trade
    mes = self.reader.createMessage(mes_trade)
AttributeError: 'Reader' object has no attribute 'createMessage'

======================================================================
ERROR: test_reader_read_line (reader_test.TestReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/joanna/Desktop/jobs/OnGoing/quantifeed/unit_test/reader_test.py", line 75, in test_reader_read_line
    self.assertEqual('F', mes.getExecType())
AttributeError: 'TradeMessage' object has no attribute 'getExecType'

----------------------------------------------------------------------
Ran 24 tests in 0.076s

FAILED (errors=5)
